src/renderer/services/Eth1ICGKeystore.ts

```diff
@@ -0,0 +1,101 @@
+import { ICGKeystore } from './interface';
+import { Keypair } from '@chainsafe/bls/lib/keypair';
+import * as fs from 'fs';
+import { PrivateKey } from '@chainsafe/bls/lib/privateKey';
+import { PublicKey } from '@chainsafe/bls/lib/publicKey';
+const eth1WalletProvider = require('ethereumjs-wallet');
+const bech32 = require('bech32');
```

Comment:
don't use require import.

Reply:
There is not `@types/ethereumjs-wallet`, but there are `@types/bech32` so it is changed

Reply:
ethereumjs-wallet is written in typescript so the library itself already contains typescript